helpers.py

```diff
@@ -27,25 +25,19 @@ def assert_if_user_is_authorized(browser):
 
 
 def get_math_function_value(x: int | str) -> str:
-    """Calculate math expression, which value
-    will be used to solve captcha
-    """
+    """Calculate math expression, which value will be used to solve captcha."""
     return str(log(abs(12 * sin(int(x)))))
 
 
 def open_browser_page(link):
-    """Call Chrome browser,
-    then open link in browser
-    """
+    """Call Chrome browser, then open link in browser."""
```

Comment:
Кстати, согласно документации, вызов `Chrome()`
> Creates a new instance of the chrome driver. Starts the service and then creates new instance of chrome driver.

Стало быть,
```suggestion
    """Create Chrome browser instance, then open link in browser."""
```